docs/changelog.md

```diff
@@ -4,9 +4,20 @@ tocMaxDepth: 2
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.97 (2022-05-10)
+
+### Features
+
+- Added the `--public` option to the `push` command. With this option, you can upload OpenAPI definitions and make them publicly accessible.
```

Comment:
```suggestion
- Added the `--public` option to the `push` command. With this option, you can upload OpenAPI definitions and make them publicly accessible.
- Fixed `preview-docs` hot reload.
- Changed assertions syntax to this pattern: `assert/{assert name}`.
```
Added some suggestions for the things that should be included in this release (I think).